.pending/improvements/gaia/3893-Improve-gaiacli-tx-sign-command

```diff
@@ -0,0 +1,4 @@
+#3893 Improve `gaiacli tx sign` command
+  * Add shorthand flags -a and -s for the account and sequence numbers respectively
```

Comment:
I don't think you need to indent manually the list of sub items. Either way, not a big one